src/Sekiban.Infrastructure.Cosmos/SekibanAzureOption.cs

```diff
@@ -5,19 +5,21 @@ namespace Sekiban.Infrastructure.Cosmos;
 public record SekibanAzureOption
 {
     public string Context { get; init; } = SekibanContext.Default;
+
     public string CosmosEventsContainer { get; init; } = "events";
     public string CosmosEventsContainerDissolvable { get; init; } = "dissolvableevents";
     public string CosmosItemsContainer { get; init; } = "items";
     public string CosmosItemsContainerDissolvable { get; init; } = "dissolvableitems";
 
-    public string? BlobConnectionString { get; init; }
-    public string? BlobConnectionStringName { get; init; }
-
-    public string? CosmosConnectionString { get; init; }
-    public string? CosmosConnectionStringName { get; init; }
     public string? CosmosEndPointUrl { get; init; }
     public string? CosmosAuthorizationKey { get; init; }
-    public string? CosmosDatabase { get; init; }
+    public string CosmosConnectionStringName { get; init; } = "SekibanCosmos";
+    public string? CosmosConnectionString { get; init; }
+    public string CosmosDatabase { get; init; } = "SekibanDb";
+
+    public string BlobConnectionStringName { get; init; } = "SekibanBlob";
```

Comment:
I think this is good opportunity to make this as a public const. (Since it was use twice as literal string)